### DISCUSSION TITLE
Icon request: fa-language-alt (as in FA 4.7)

### DISCUSSION THREAD

**Discussion by ajtruckle** [FortAwesome/Font-Awesome#21222] [Icon Requests]
Hi

I don’t like the version 5 if the fa-language icon. Can you please reinstate the 4.7.0 version as an alternative. 

Also why have icons like:

```
fa fa-cc-paypal
fa fa-facebook-square
fa fa-youtube-square
```
Not only changed but are now **fab** instead of **fa**? By upgrading to version five I have lost icons on my site and have had to revert to 4.7. 

Thank you.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

> fa fa-cc-paypal

Doesn't look changed to me:
- https://fontawesome.com/icons/cc-paypal?style=brands
- https://fontawesome.com/v4.7.0/icon/cc-paypal

If the rectangular shape is quite different, it is because we have a new grid system

> fa fa-facebook-square

Doesn't look changed to me:
- https://fontawesome.com/icons/facebook-square?style=brands
- https://fontawesome.com/v4.7.0/icon/facebook-square

It is about the border radius? We are using a new border radius for *square* icons


> fa fa-youtube-square

That's because of https://www.youtube.com/yt/about/brand-resources/#logos-icons-colors

Ref: #11457

> Not only changed but are now fab instead of fa? By upgrading to version five I have lost icons on my site and have had to revert to 4.7.

FA5 changed a lot of things and we weren't able to keep it backward compatible. Please take a look at https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4

**Comment by ajtruckle** [comment]
The first two:

`<i class="fab fa-cc-paypal"></i>`
`<i class="fab fa-facebook-square"></i>`

They are now `fab` and not `fa` as I see details in the notes. But this change to `fab` has broken the menu icons plugin for WordPress and none of the `fab` icons can be used at the moment.

I take your point on the YouTube icon but again, since they are now `fab` prefixed I currently can't use them.

And my comment still stands about `fa fa-language`. I don't like it at all. The previous version is a square shape and I kindly ask that you re-instate it as an alternative. By changing it to a rectangle instead of a square has pushed out all the sizing of some of my controls and they no longer look as nice on smaller devices. Please re-instate that language icon.

**Comment by tagliala** [comment]
> They are now fab and not fa 

Yes, we had to make choices. Again, sorry for the breaking change

Ref: https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4

> But this change to fab has broken the menu icons plugin for WordPress and none of the fab icons can be used at the moment.

The plugin requires an updated version compatible with Font Awesome 5

> The previous version is a square shape and I kindly ask that you re-instate it as an alternative.

This request is open for upvotes

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
